packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/CancelTransaction/CancelTransaction.tsx

```diff
@@ -0,0 +1,122 @@
+        return;
+    }
+
+    if (composedCancelTx.outputs.length !== 1) {
+        return null;
+    }
+
+    const output = composedCancelTx.outputs[0];
+
+    const feePerByte = new BigNumber(composedCancelTx.feePerByte);
+    const fee = formatNetworkAmount(composedCancelTx.fee, tx.symbol);
+
+    return (
+        <Card fillType="flat" paddingType="none">
+            <Row justifyContent="space-between" margin={spacings.md}>
+                <Text typographyStyle="highlight">
+                    <Translation id="TR_CANCEL_TX_HEADER" />
+                </Text>
+
+                <Text typographyStyle="hint">
+                    <TrezorLink
+                        variant="nostyle"
+                        href={HELP_CENTER_CANCEL_TRANSACTION}
+                        icon="arrowUpRight"
+                    >
+                        <Translation id="TR_LEARN_MORE" />
+                    </TrezorLink>
+                </Text>
+            </Row>
+
+            <Divider margin={spacings.zero} />
+
+            <Column gap={spacings.md} margin={spacings.md}>
+                <Text typographyStyle="hint">
+                    <Translation id="TR_CANCEL_TX_NOTICE" />
+                </Text>
+
+                <InfoItem
+                    direction="row"
+                    label={
+                        <Row gap={spacings.md}>
+                            <Translation id="TR_CANCEL_TX_FEE" />
+                            <Text variant="tertiary">
+                                {feePerByte.toFormat(2)}&nbsp;
+                                {getFeeUnits(networkType)}
+                            </Text>
+                        </Row>
+                    }
+                    typographyStyle="body"
+                    variant="default"
+                >
+                    <Column gap={spacings.md} alignItems="flex-end">
+                        <FormattedCryptoAmount
+                            disableHiddenPlaceholder
+                            value={fee ?? undefined}
+                            symbol={tx.symbol}
+                        />
+                        <Text variant="tertiary" typographyStyle="label">
+                            <FiatValue
+                                disableHiddenPlaceholder
+                                amount={fee ?? '0'}
+                                symbol={tx.symbol}
+                            />
+                        </Text>
+                    </Column>
+                </InfoItem>
+
+                <Divider
+                    orientation="horizontal"
+                    margin={{ top: spacings.xs, bottom: spacings.xs }}
+                />
+
+                <InfoItem
+                    direction="row"
+                    label={<Translation id="TR_CANCEL_TX_RETURN_TO_YOUR_WALLET" />}
+                    typographyStyle="body"
+                    variant="default"
+                >
+                    <Column gap={spacings.md} alignItems="baseline">
```

Comment:
Sorry, I should have written comments for both cases instead of saying "same below", it can get overlooked easily..
```suggestion
                    <Column gap={spacings.md} alignItems="flex-end">
```